### DISCUSSION TITLE
Android app wifi (internal ip) external ip (to use on mobile) settings?

### DISCUSSION THREAD

**Discussion by randomerrorgenerator** [clangen/musikcube#455] [Ideas]
not sure how to phrase this , but i've been able to get this to work pretty nicely.

Internally on the home wifi (192.168.x.x) , but also trying to set this up so i can listen at work, i've made it so i can turn off my phones wifi and use mobile data on android, and it connects (75.87.x.x) , and plays, but when i turn wifi back on the android app can't see musikcube, unless i use its internal ip not sure if this can really be done, but would be nice to set in 2 ip's and have it use which one it can connect to.

**Comment by extrowerk** [comment]
I use tailscale so i can connect to my NAS via 192.168.x.x even away from home. You could try it.

**Comment by Joshfindit** [comment]
If I’m interpreting this issue correctly, I think that Plex does something that would help:

In the Plex apps you can specify multiple IPs/hostnames for the same server. This allows a user to specify internal and external IPs independent of the server config (especially handy for external access through port forwarding and VPNs)

**Comment by tomasmez** [comment]
you can setup an openvpn server to do it secure or expose the ports 7905/7906 at your router
Either way you need a static ip or use a dynamic dns name service
